index.js

```diff
@@ -19,5 +19,24 @@ module.exports = function(source) {
         this.cacheable();
     }
 
+    // if force sourcemap requested embed inline sources (helps resolve issues with
+    // transpiled languages like typescript
+    if (userOptions['force-sourcemap']) {
```

Comment:
This is more of a hack as there may actually be standalone source maps alongside, but they don't seem to work for reasons I forget. This pr allows forcing an inline sourcemap to be appended, I don't think its actually a good long-term solution